keyboards/handwired/dactyl_manuform/5x6_68/5x6_68.h

```diff
@@ -0,0 +1,66 @@
+// Copyright 2023 Kostas Pagratis (@kpagratis)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "quantum.h"
+
+#define XXX KC_NO
+
+#define LAYOUT_5x6_68( \
+         L00, L01, L02, L03, L04, L05,      \
+         L10, L11, L12, L13, L14, L15,      \
+         L20, L21, L22, L23, L24, L25,      \
+         L30, L31, L32, L33, L34, L35,      \
+         L40, L41, L42, L43,                \
+                             L50, L51, L52, \
+                             L53, L54, L55, \
+    \
+         R00, R01, R02, R03, R04, R05, \
+         R10, R11, R12, R13, R14, R15, \
+         R20, R21, R22, R23, R24, R25, \
+         R30, R31, R32, R33, R34, R35, \
+                   R42, R43, R44, R45, \
+    R50, R51, R52, \
+    R53, R54, R55 \
+) { \
+    { L00, L01, L02, L03, L04, L05 }, \
+    { L10, L11, L12, L13, L14, L15 }, \
+    { L20, L21, L22, L23, L24, L25 }, \
+    { L30, L31, L32, L33, L34, L35 }, \
+    { L40, L41, L42, L43, XXX, XXX }, \
+    { L50, L51, L52, L53, L54, L55 }, \
+\
+    { R00, R01, R02, R03, R04, R05 }, \
+    { R10, R11, R12, R13, R14, R15 }, \
+    { R20, R21, R22, R23, R24, R25 }, \
+    { R30, R31, R32, R33, R34, R35 }, \
+    { XXX, XXX, R42, R43, R44, R45 }, \
+    { R50, R51, R52, R53, R54, R55 } \
+}
+
+/*
+#define LAYOUT_5x6_68_PRETTY( \
+    L00, L01, L02, L03, L04, L05,                          R00, R01, R02, R03, R04, R05, \
+    L10, L11, L12, L13, L14, L15,                          R10, R11, R12, R13, R14, R15, \
+    L20, L21, L22, L23, L24, L25,                          R20, R21, R22, R23, R24, R25, \
+    L30, L31, L32, L33, L34, L35,                          R30, R31, R32, R33, R34, R35, \
+    L40, L41, L42, L43,                                              R42, R43, R44, R45, \
+                        L50, L51, L52,                R50, R51, R52, \
+                        L53, L54, L55,                R53, R54, R55 \
+) { \
+    { L00, L01, L02, L03, L04, L05 }, \
+    { L10, L11, L12, L13, L14, L15 }, \
+    { L20, L21, L22, L23, L24, L25 }, \
+    { L30, L31, L32, L33, L34, L35 }, \
+    { L40, L41, L42, L43, XXX, XXX }, \
+    { L50, L51, L52, L53, L54, L55 }, \
+\
+    { R00, R01, R02, R03, R04, R05 }, \
+    { R10, R11, R12, R13, R14, R15 }, \
+    { R20, R21, R22, R23, R24, R25 }, \
+    { R30, R31, R32, R33, R34, R35 }, \
+    { XXX, XXX, R42, R43, R44, R45 }, \
+    { R50, R51, R52, R53, R54, R55 } \
+}
+*/
```

Comment:
This file should be deleted (assuming the `matrix` data in info.json file is correct)